ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/host-sets/host-set/hosts/index.hbs

```diff
@@ -59,51 +59,47 @@
                   {{! Link to a different subroute for dynamic/plugin hosts }}
                   <LinkTo
                     @route='scopes.scope.host-catalogs.host-catalog.host-sets.host-set.hosts.host'
-                    @model={{host.id}}
+                    model={{host.id}}
                   >
                     {{host.displayName}}
                   </LinkTo>
                 {{/if}}
-                <p>{{host.description}}</p>
-              </row.headerCell>
-              <row.cell>
+                <Hds::Text:Body @tag='p'>{{host.description}}</Hds::Text:Body>
+              </B.Td>
+              <B.Td>
                 <HostCatalogTypeBadge @model={{host}} />
-              </row.cell>
-              <row.cell>
-                <Copyable
-                  @text={{host.id}}
-                  @buttonText={{t 'actions.copy-to-clipboard'}}
-                  @acknowledgeText={{t 'states.copied'}}
-                >
-                  <code>{{host.id}}</code>
-                </Copyable>
-              </row.cell>
-              <row.cell>
+              </B.Td>
+              <B.Td>
+                <Hds::Copy::Snippet @textToCopy={{host.id}} @color='secondary'>
+                  <Hds::Text::Code>{{host.id}}</Hds::Text::Code>
```

Comment:
Why are we wrapping `Hds::Text::Code` within `Hds::Copy::Snippet` ?

Within [`Hds::Copy::Snippet` documentation,](https://helios.hashicorp.design/components/copy/snippet?tab=code) the only pattern prescribed is to use the attribute `@textToCopy` to pass the text we want the user to see and copy.

The `Hds::Copy::Snippet` does not support this pattern, in fact, it is not "yielding" anything to its content. If you try to nest content inside will be ignored. [You can check the component code](https://github.com/hashicorp/design-system/blob/main/packages/components/addon/components/hds/copy/snippet/index.hbs).

Reply:
@calcaide ahh that is correct! fix incoming